src/lib/handlers/info-command.handler.ts

```diff
@@ -0,0 +1,88 @@
+import { exec } from 'child_process';
+import * as fs from 'fs';
+import * as nodemon from 'nodemon';
+import * as path from 'path';
+import { Logger } from '../../common/logger/interfaces/logger.interface';
+import { GenerateCommandArguments } from '../../common/program/interfaces/command.aguments.interface';
+import { CommandHandler } from '../../common/program/interfaces/command.handler.interface';
+import { GenerateCommandOptions } from '../../common/program/interfaces/command.options.interface';
+import { ConfigurationService } from '../../core/configuration/configuration.service';
+import { ColorService } from '../../core/logger/color.service';
+import { LoggerService } from '../../core/logger/logger.service';
+import { nestWords } from '../../core/utils/nest-ascii';
+import osName = require('os-name');
+
+
+export class InfoCommandHandler implements CommandHandler {
+
+    private logger: Logger;
+
+    public execute(
+        args: GenerateCommandArguments,
+        options: GenerateCommandOptions,
+        logger: Logger
+    ): Promise<void> {
+        LoggerService.setLogger(logger);
+        this.logger = logger;
+        logger.debug(ColorService.blue('[DEBUG]'), 'execute serve command');
+        return ConfigurationService.load()
+            .then(async () => {
+                const nodeVersion = process.version + ' ' + ((<any>process).release.lts || '');
+                const nestVersion = await this.getNestVersion();
+                const cliVersion = await this.getCliVersion();
+                const npmVersion = await this.getNpmVersion();
+                logger.info(ColorService.yellow(nestWords));
+                logger.info(ColorService.green('[System Information]'))
+                logger.info(this.concatenate('OS Version', osName()));
```

Comment:
Couldn't [node OS API](https://nodejs.org/api/os.html#os_os_release) work for this ?  
There is `os.release() os.type() os.platform()` I never used it though (introduced in v0.3.3).

Reply:
Its not specific enough, believe me I tried for 2 hours to get it to work.